bugs.js

```diff
@@ -99,10 +108,22 @@ const getBugzilla = async (website, bugzillaKey) => {
  * @returns an Object with a webcompatResult and a criticalsResult properties
  *          that correspond to each query
  */
-const getWebcompat = async (website, githubKey) => {
+const getWebcompat = async (website, githubKey, minDate, maxDate) => {
     const spaced = website.replace(/\./g, " ");
-    const webcompatQuery = `https://github.com/search?q=${spaced}+in%3Atitle+repo%3Awebcompat%2Fweb-bugs%2F+state%3Aopen+label:engine-gecko&type=Issues`;
-    const criticalsQuery = `https://github.com/webcompat/web-bugs/issues?q=${spaced}+in%3Atitle+repo%3Awebcompat%2Fweb-bugs%2F+is%3Aopen+label%3Aseverity-critical+label:engine-gecko`;
+    let state = "+state:open"
```

Comment:
```suggestion
    let state = encodeURIComponent("+state:open");
```
encodeURIComponent just to be on the safe side?

Reply:
That will actually break the API calls, because this is written to them as well, and they will double-encode that way (but I'm ok with encoding the links written out to the CSV HYPERLINK values, if we'd like).

Reply:
Ah, OK then. It's not a big deal either way.